src/blockstore/meta.zig

```diff
@@ -0,0 +1,344 @@
+const std = @import("std");
+const sig = @import("../lib.zig");
+
+const Allocator = std.mem.Allocator;
+
+const BitFlags = sig.utils.bitflags.BitFlags;
+const Shred = sig.shred_collector.shred.Shred;
+const CodingShred = sig.shred_collector.shred.CodingShred;
+const Slot = sig.core.Slot;
+const SortedSet = sig.utils.collections.SortedSet;
+
+/// The Meta column family
+pub const SlotMeta = struct {
+    // allocator: Allocator,
```

Comment:
should delete the allocator mentions that are commented out if we arent using them (are throughout the file too)

Reply:
f88f48dce43ee03831e536c2cc67e810451e25ff